### PR TITLE
setup: perform all pyenv setup in a single place

### PR DESCRIPTION
**Pull request by gregjhogan**
The `pyenv` env setup isn't part of openpilot so I don't think it should be in `openpilot_env.sh`

also, the whole `pyenv init --path` thing seems to have only been an issue running `pyenv update` the first time after they transitioned to recommending ~/.profile (I am not seeing any issue on a fresh install or updating from a recent version)

### CODE REVIEW

File: update_requirements.sh [hunk 1]
```diff
@@ -3,11 +3,21 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 cd $DIR
 
+RC_FILE="${HOME}/.$(basename ${SHELL})rc"
```
Comment (adeebshihadeh) [inline update_requirements.sh:6]: this isn’t always the case. mac defaults to ~/.bash_profile

Comment (gregjhogan) [reply update_requirements.sh:6]: .bash_profile normally calls .bashrc so I thought this is a superset, didn't realize mac doesn't do this :(

**Comment by adeebshihadeh** [comment]
Still want this?

**Comment by gregjhogan** [comment]
yeah I will finish it up, it seems weird to me to have pyenv environment config in `openpilot_env.sh` since it applies much wider than just openpilot

**Comment by gregjhogan** [comment]
This should be good now once CI passes (assuming CI sufficiently tests the setup)

**Comment by pd0wm** [comment]
The openpilot env still sets some variables on Mac. Why do you want to remove it?

**Pull request merged**